azure/packages/azure-client/src/AzureFunctionTokenProvider.ts

```diff
@@ -12,6 +12,11 @@ import { AzureMember } from "./interfaces";
  * Token Provider implementation for connecting to an Azure Function endpoint for
```

Comment:
This comment should be merged with the new one below. I.e.
```typescript
 * Token Provider implementation for connecting to an Azure Function endpoint for
 * Azure Fluid Relay token resolution.
 * 
 * @deprecated 1.2.0, This API will be removed in 2.0.0.
 * No replacement since it is not expected anyone will use this token provider as is.
 * See https://github.com/microsoft/FluidFramework/issues/13693 for context
```